website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+You can check out the [**video**](https://www.youtube.com/watch?v=eUa9ficyOPg)
+
+### How to implement FLINT?
+Below given are instructions for quick installation. 
+
+**NOTE:** These instructions are for building the FLINT on Windows using Visual Studio 2017, or Visual Studio 2019.
+
+1] Install CMake: 
+```
+CMake-3.15.2-win64-x64.msi
+```
+2] Using *vcpkg* to install required libraries
+
+A fork of a Vcpkg repository has been created for the FLINT required libraries. To build these libraries you can use the following process:
+
+- Clone the Vcpkg repository: https://github.com/moja-global/vcpkg
+- Start a command shell in the Vcpkg repository folder and use the following commands:
+```
+# bootstrap
+bootstrap-vcpkg.bat
+
+# install packages
+vcpkg.exe install boost-test:x64-windows boost-program-options:x64-windows boost-log:x64-windows turtle:x64-windows zipper:x64-windows poco:x64-windows libpq:x64-windows gdal:x64-windows sqlite3:x64-windows boost-ublas:x64-windows
+```
+- Start a command shell in your FLINT repository folder after this is done. To construct the Visual Studio solution, use the following commands:
+
+```
+# Create a build folder under the Source folder
+cd Source
+mkdir build
+cd build
+
+# now create the Visual Studio Solution (2019)
+cmake -G "Visual Studio 16 2019" -DCMAKE_INSTALL_PREFIX=C:/Development/Software/moja -DVCPKG_TARGET_TRIPLET=x64-windows -DENABLE_TESTS=OFF -DENABLE_MOJA.MODULES.ZIPPER=OFF -DCMAKE_TOOLCHAIN_FILE=c:\Development\moja-global\vcpkg\scripts\buildsystems\vcpkg.cmake ..
+
+# OR Visual Studio Solution (2017)
+cmake -G "Visual Studio 15 2017" -DCMAKE_INSTALL_PREFIX=C:/Development/Software/moja -DVCPKG_TARGET_TRIPLET=x64-windows -DENABLE_TESTS=OFF -DENABLE_MOJA.MODULES.ZIPPER=OFF -DCMAKE_TOOLCHAIN_FILE=c:\Development\moja-global\vcpkg\scripts\buildsystems\vcpkg.cmake ..
+```
+3] **Install Moja Libraries** 
+
+The Visual Studio moja solution can be used to install compiled versions of the Moja libraries. Set the CMAKE variable **'CMAKE INSTALL PREFIX'** to your installation path (for example, *"C:/Development/Software/moja"* ).
+
+4] Using CMake, make changes to the Visual Studio Solution.
+
+- Open the CMake graphical user interface.
+- Click **"Browse Build..."** and select the folder you created previously (i.e. ```C:Developmentmoja-globalFLINTSourcebuild``` in the **"Where to build the binaries"** area) 
+- If the 'Where is the source code:' field does not update, make the necessary changes.
+- You should now be able to update any CMake settings (for example, ENABLE flags like ENABLE TESTS), then click **“Configure”** - there should be no issues presuming all libraries and essential software have been installed. 
+- Now click **"Generate,"** and the Solution should be ready to load into Visual Studio with the changes.
+
+### Other Useful Tools :
```

Comment:
@sarthakkundra - when we have a a blog on the FLINTexample analysis it'd be great to add a  'Next steps' section at the end that links to what to do after installation.

Reply:
Okay I will add the next steps on how to install the other useful tools

Reply:
No need @ruchip16 - I was only alerting @sarthakkundra to a potential future improvement. Your work so far is great!

Reply:
Okay, thank you!